README.md

```diff
@@ -1,3 +1,3 @@
-# Alpakka samples
+# Pekko Connectors samples
 
-[Alpakka documentation](https://docs.akka.io/docs/alpakka/)
+Samples for [Pekko Connectors](https://pekko.apache.org/docs/pekko-connectors/current/).
```

Comment:
![image](https://github.com/apache/incubator-pekko-connectors-samples/assets/501740/7ba58bd6-85e2-4603-a56f-102bd8aed792)

Reply:
https://pekko.apache.org/docs/pekko-connectors/current/ works for me